src/shared/condition.c

```diff
@@ -177,6 +178,27 @@ static int condition_test_user(Condition *c) {
         return id == getuid() || id == geteuid();
 }
 
+static int condition_test_control_group_controller(Condition *c) {
+        int r;
+        CGroupMask system_mask, wanted_mask;
+
+        assert(c);
+        assert(c->parameter);
+        assert(c->type == CONDITION_CONTROL_GROUP_CONTROLLER);
+
+        r = cg_mask_supported(&system_mask);
+        if (r < 0) {
+                log_error_errno(r, "Failed to determine supported controllers: %m");
+                return -EBADF;
+        }
+
+        r = cg_mask_from_string(c->parameter, &wanted_mask);
+        if (r < 0)
+                return 0;
```

Comment:
it makes sense to eat up parsing errors here i think, but maybe add a log_debug_errno() and say so, to make this easier to debug